mne/viz/raw.py

```diff
@@ -559,14 +563,19 @@ def plot_raw_psd(raw, tmin=0., tmax=np.inf, fmin=0, fmax=np.inf, proj=False,
         If True, transform data to decibels.
     average : bool
         If False, the PSDs of all channels is displayed. No averaging
-        is done and parameters area_mode and area_alpha are ignored.
+        is done and parameters area_mode and area_alpha are ignored. When
+        False, it is possible to paint an area (hold left mouse button and
+        drag) to plot a topomap.
     show : bool
         Show figure if True.
     n_jobs : int
         Number of jobs to run in parallel.
     line_alpha : float | None
         Alpha for the PSD line. Can be None (default) to use 1.0 when
         ``average=True`` and 0.1 when ``average=False``.
+    spatial_colors : bool
+        Whether to use spatial colors. Only works when average=False. Defaults
+        to False.
```

Comment:
This default is no longer correct. You could say "Defaults to False when average=False, but this will change to True in 0.14"